.github/workflows/release-notify.yml

```diff
@@ -0,0 +1,16 @@
+name: Close issue asking for release
+
+on:
+  issues:
+    types: [opened]
+  issue_comment:
+    types: [created]
+    
+
+jobs:
+  action-test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: niklasmerz/release-notify@master
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Won't we need infra to add the secret?

Reply:
https://docs.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token

I did not find an option to disable it in my orgs. Not sure though.